book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals << rental
+    rental.book = self
+  end
```

Comment:
- [ ] Remember when we are creating a rental between the book and the person, we actually need to create a new rental. So It will be great to initialize a new rental class and pass in the date, self(the book itself) and the person hence achieving the many-to-many relationship. Check the hint below ⬇️ 
```ruby
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```